server/command/daily_summary.go

```diff
@@ -59,6 +60,12 @@ func (c *Command) dailySummary(parameters ...string) (string, error) {
 			return err.Error(), err
 		}
 		return dailySummaryResponse(dsum), nil
+	case "all":
```

Comment:
This is for debugging purposes. The functionality is limited to admins. I can't see it being useful to admins, as it is essentially "Run this job now even though it will run in the next 10-15 minutes." Also, it would only run for users that are in the correct time window anyway. Maybe just take it out?

Reply:
Removed this block